scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncherInitializer.java

```diff
@@ -274,6 +275,20 @@ public int getReplicationIndex() {
     }
 
     /**
+     * @param genericInformation the generic information of this task
+     */
+    public void setGenericInformation(Map<String, String> genericInformation) {
```

Comment:
It were nice to have it only set a construction time. What about the executable container, doesn't task related information belong more into it the executable container rather than the tasklauncher?